drivers/platform/maxim/maxim_gpio_irq.c

```diff
@@ -360,5 +377,6 @@ const struct no_os_irq_platform_ops max_gpio_irq_ops = {
 	.trigger_level_set = (int32_t (*)())max_gpio_irq_trigger_level_set,
 	.global_enable = (int32_t (*)())max_gpio_irq_global_enable,
 	.global_disable = (int32_t (*)())max_gpio_irq_global_disable,
+	.set_priority = (int32_t (*)())(max_irq_set_priority),
```

Comment:
just to comply with the reset of the APIs, I would suggest renaming this with max_gpio_irq_set_priority